src/CliCommand/SoxCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace IndieHD\AudioManipulator\CliCommand;
+
+use IndieHD\AudioManipulator\CliCommand\SoxCommandInterface;
+
+class SoxCommand extends CliCommand implements SoxCommandInterface
+{
+    private $name = 'sox_command';
+
+    protected $binary = 'sox';
```

Comment:
Make this an over-writable property in the base class

Reply:
Same question here; should the value be initialized an empty string?